caom2-artifact-resolvers/src/main/java/ca/nrc/cadc/caom2/artifact/resolvers/GeminiResolver.java

```diff
@@ -85,7 +85,7 @@ private String getPath(URI uri) {
         String fileName = path[1];
         String fileType = FILE_URI;
         String[] fileNameParts = fileName.split("\\.");
-        if (fileNameParts[1].equals("jpg")) {
+        if (fileName.endsWith("jpg")) {
```

Comment:
Should be ".jpg" and you can delete line 87

Reply:
Oops! Will add the '.'
fileNameParts[0] is used to form the new file name, no?